project/Maven.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package akka
+
+import java.io.File
+
+import sbt._
+
+object Maven {
+  val mvn = config("maven")
+
+  val projects = settingKey[Seq[File]]("Projects which should be built by calling maven")
+  val defaultCommands = settingKey[Seq[String]]("Commands to be called on each maven project")
+  val javaHomeOverride = settingKey[Option[File]]("JAVA_HOME to be used while building projects with maven")
+  
+  val testAll = taskKey[Unit]("Test all maven projects (as listed in projectsToTest)")
```

Comment:
this is the task so projectsToTest sounds a bit off, it actually runs all tests (for all `projects`). 
Maybe `mvn test` => `mvnTest` would make the most sense here then.